### PR TITLE
Audit typescript props and add a demo for VictoryCursorContainer

### PR DESCRIPTION
**Pull request by maddles**
- Enforce number from cursorLabel function
- Added VictoryCursorContainer demo

### CODE REVIEW

File: packages/victory-cursor-container/src/index.d.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { VictoryContainerProps, CursorData } from "victory-core";
 export interface VictoryCursorContainerProps extends VictoryContainerProps {
   cursorComponent?: React.ReactElement;
   cursorDimension?: "x" | "y";
-  cursorLabel?: (point: CursorData) => void;
+  cursorLabel?: (point: CursorData) => number;
```
Comment (boygirl) [inline packages/victory-cursor-container/src/index.d.ts:7]: This is incorrect. `cursorLabel` should be able to return a string, too.

Comment (maddles) [reply packages/victory-cursor-container/src/index.d.ts:7]: Shall I do string | number, or is there a situation where void should be accepted too?

Comment (boygirl) [reply packages/victory-cursor-container/src/index.d.ts:7]: Yeah, folks sometimes return null to render no label for a given data point. Let's leave this one as flexible as possible.

**Comment by boygirl** [comment]
Approved!

**Pull request merged**